### ISSUE TITLE
feat: Make iPhone widget available on Mac

### ISSUE THREAD

**Issue by ya-d** [twofas/2fas-ios#92]
### Feature description

It would be great if 2FAS widgets could be used on Mac which is possible since iOS 17 / macOS 14. Maybe it's blocked by using NSFileProtection. See https://developer.apple.com/documentation/widgetkit/creating-a-widget-extension. Feature description: https://support.apple.com/de-de/guide/mac-help/mchl52be5da5/mac.

### Motivation

Great way to quickly access tokens without touching your phone. Especially since the App itself is not yet available on macOS (#36). Thanks for considering!

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by adocyn** [comment]
Hi @ya-d. We've tested that and yes, we would need to disable "user is authenticated" protection and in the end it doesn't work very reliable. Had high hopes for that too. 
So we're not planning that right now, but we'll see what upcoming iOS version brings in this topic.
